packages/cardhost/app/templates/components/card-creator.hbs

```diff
@@ -20,34 +20,100 @@
 <br>
 
 {{#if this.card}}
-  <CardRenderer
-    @card={{this.card}}
-    @format="isolated"
-    @mode="schema"
-    @removeField={{action this.removeField}}
-    @setNeededWhenEmbedded={{action this.setNeededWhenEmbedded}}
-    @setFieldName={{action this.setFieldName}}
-    @setPosition={{action this.setPosition}}
-  >
-    <div class="card-creator--field-btn-wrapper">
-      {{#unless this.cardstackSession.isAuthenticated}}
-        (Must be logged in to save the card.)
-      {{/unless}}
-      <button {{on "click" this.save}}
-        data-test-card-creator-save-btn
-        disabled={{not this.cardstackSession.isAuthenticated}}
-      >
-        Save Card
-      </button>
+  <div class="card-manipulator--container">
+    {{!-- Left edge --}}
+    <div class="card-manipulator--left-edge">
+      {{!-- Profile sidebar --}}
+      <div class="card-manipulator--left-edge--profile">
+        <div class="profile icon"></div>
+        <div class="settings icon active"></div>
+        <div class="search icon"></div>
+      </div>
+      {{!-- Catalog fields list --}}
+      <div class="card-manipulator--left-edge--catalog">
+        <div class="title">Card Catalog</div>
+
+        {{#each this.fieldComponents as |field|}}
+          <div
+            class="card-manipulator--left-edge--catalog-field"
+            role="button"
+            draggable="true"
+            onmousedown={{action this.beginDragging field}}
+            data-test-card-add-field-draggable={{field.type}}
```

Comment:
ditto from the comment in `card-schema-updator.hbs` about using `{{on}}` instead of hand wiring `window.addEventListener()` (and having to deal with the bookkeeping of `window.removeEventListener()` when the component is destroyed, or in your case when the drag is done).